streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -230,6 +244,111 @@ public void testTotalMetricDoesntDecrease() {
             assertEquals(i, Math.round(totalMetric.measurable().measure(config, time.milliseconds())));
             sensor.record(latency, time.milliseconds());
         }
+    }
+
+    @Test
+    public void shouldAddAmountRateAndSum() {
+        final MockTime time = new MockTime(0);
+
+        addAmountRateAndTotalMetricsToSensor(sensor, group, tags, metricNamePrefix, description1, description2);
```

Comment:
Nit: I think it is confusing to import methods under test (makes it unclear if this is a "setup" method -- might be better to not use static imports) ?